_posts/2020-11-30-dataops-dynamodb-to-snowflake.md

```diff
@@ -0,0 +1,378 @@
+---
+layout: post
+title: Batch ELT from AWS DynamoDB to Snowflake
+date: 2020-11-30
+tags: snowflake dynamodb aws data elt s3
+author: Paul Symons
+image: img/blog/dynamodb-to-snowflake/logo.jpg
+---
+<style type="text/css">
```

Comment:
Interesting that this custom CSS is needed here. Was there an issue with formatting the article?

Reply:
oops i left that in. i was toying with the pre-heading margins and screen width for larger screens because the default - jekyll layout controlled i think - is not very generous. with the volume of screenshots and headings I had, everything looked a bit squeezed. i'll probably pull this out pending a wider discussion about blog styling as they should probably all be consistently presented